lib/db/StudyRequestDAO.js

```diff
@@ -78,6 +78,43 @@ SELECT
     });
   }
 
+  /**
+   * Fetch all study requests.
+   *
+   * @returns {Array<Object>} array of all study request objects
+   */
+  static async all() {
```

Comment:
we should eventually add tests for `StudyRequestDAO.all`, `StudyRequestDAO.byUser` here.